cmd/podman/ps.go

```diff
@@ -564,6 +565,10 @@ func getAndSortJSONParams(containers []*libpod.Container, opts batchcontainer.Ps
 	for _, ctr := range containers {
 		batchInfo, err := batchcontainer.BatchContainerOp(ctr, opts)
 		if err != nil {
+			if errors.Cause(err) == libpod.ErrNoSuchCtr {
+				logrus.Error(err)
```

Comment:
If this is non-fatal, I think it should be `Warn` instead of `Error`.

Another way to approach this, which would allow `getAndSortJSONParams` to be less opinionated, would be to take a callback (like [`filepath.Walk`][1]).  That way the caller could explicitly decide which `BatchContainerOp` errors were fatal (e.g. "everything except `ErrNoSuchCtr`" and what to do about non-fatal errors (e.g. "submit warn-level logs").

[1]: https://golang.org/pkg/path/filepath/#Walk

Reply:
> Another way to approach this, which would allow `getAndSortJSONParams` to be less opinionated...

I just realized `getAndSortJSONParams` is in `cmd/podman/ps.go`, not in libpod, so no need for a n unopinionated API.  I still think we want to `Warn` instead of `Error`, though.

Reply:
+1 on the Warning if we're just going to continue.

Reply:
Its funny I started with Warn and went to Error.  But if consensus is warn I am fine with it, especially when it is an expected, while rare condition.